basic-examples/quoting/README.md

```diff
@@ -1,15 +1,16 @@
-# Expressions
+# Quoting
 
 ## Workflow Description
 
-This workflow demonstrates example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+This workflow demonstrates example expressions with quotes as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/).
+There are multiple types of quoting, and quotes must pass through YAML's syntax. Look at this example to learn about the various ways to determine what would work best for your use case.

```

Comment:
Looking at the linked-to documentation (not to mention the workflow input), I think we bury the lead.  🙃

That is, I think the [_last_ example](https://github.com/arcalot/arcaflow-workflows/blob/a3f10f67fa1713ed86fa86245488b3212decdce1/basic-examples/quoting/workflow.yaml#L21-L22) is the best one.  And, yet, especially in the documentation, we have _one_ sentence describing "string values" before we go on for **_pages_** on special characters, escaping, and quoting.  Now, granted, that stuff is very important (and hard to explain succinctly), but, if the reader could just skip to the end, using "raw" strings would potentially make comprehending all the rest of it unnecessary.  🥴

_[So, this comment isn't really actionable in this PR, but we should consider presenting "best practice" first (which I might suggest would be -- especially, if you need to embed quotation marks in your strings -- to consider using YAML "Block Flow Scalar" format with ARCA "raw" strings)...and, **then** we can point out that, if that's not suitable, we have these other four or five ways to approach your problem.  🙂]_